src/classes/VOL_CTRL_VolunteersJobListingFS.cls

```diff
@@ -29,6 +29,15 @@
 */
 
 global virtual with sharing class VOL_CTRL_VolunteersJobListingFS {
+    /**
+     * Hard coding the subquery limit to 200 to prevent an error being displayed when
+     * attempting to access the list of child records when more than 200 are present
+     * https://success.salesforce.com/issues_view?id=a1p300000008YGjAAM&title=invalid-query-locator-error-occurs-when-iterating-child-objects-in-parent-child-relationship-query-result
+     * NOTE: In a test environment we are not hitting this error until we have 250 child records
+     * but our customers are running into this at 200 likely due to more fields being populated and returned
+     */
+    @TestVisible
+    private static Integer SUBQUERY_LIMIT = 200;
```

Comment:
If we're not attempting to communicate class state, should our constant be `static final`?

Reply:
Yea I totally thought I did, having one of those days, thanks for catching this!